examples/bc5cdr.py

```diff
@@ -68,32 +69,46 @@
     "bigbio": "http://www.biocreative.org/media/store/files/2016/CDR_Data.zip",
 }
 
+_SUPPORTED_TASKS = ["NER", "NED", "RE"]
+_SOURCE_VERSION = datasets.Version("01.05.16")
+_BIOBIO_VERSION = datasets.Version("1.0.0")
+
+
+@dataclass
+class BigBioConfig(datasets.BuilderConfig):
+    """BuilderConfig for BigBio."""
+
+    schema: str = None  # options = (source|bigbio)
+    task_id: str = None  # e.g, bioasq10b
+
+
 class Bc5cdrDataset(datasets.GeneratorBasedBuilder):
     """
     BioCreative V Chemical Disease Relation (CDR) Task.
     """
 
-    VERSION = datasets.Version("01.05.16")
+    DEFAULT_CONFIG_NAME = "bc5cdr_source"
 
     BUILDER_CONFIGS = [
-        datasets.BuilderConfig(
-            name="source", version=VERSION, description="Source format",
+        BigBioConfig(
+            name="bc5cdr_source",
+            version=_SOURCE_VERSION,
+            description="BC5CDR source schema",
+            schema="source",
+            task_id="bc5cdr",
         ),
-        datasets.BuilderConfig(
-            name="bigbio",
-            version="1.0.0",
-            description="BigScience biomedical hackathon schema",
+        BigBioConfig(
+            name="bc5cdr_bigbio",
+            version=_BIOBIO_VERSION,
+            description="BC5CDR simplified BigBio schema",
+            schema="bigbio_kb",
+            task_id="bc5cdr",
```

Comment:
my impression when reading this config variable is that it would be something like "QA" or "NER" ... should we rename this to `dataset_name` maybe? if the goal is to capture things like the year in a BioASQ task then we could take it even further and have a BioASQ config that inherits from BigBioConfig and has a default value for a "years" field. that would allow it to be treated the same as all BigBio Configs but also give users the power to control which years are used.